fe/fe-core/src/main/java/com/starrocks/catalog/EsTable.java

```diff
@@ -71,7 +71,7 @@ public class EsTable extends Table {
     public static final String USER = "user";
     public static final String PASSWORD = "password";
     public static final String INDEX = "index";
-    public static final String TYPE = "type";
+    public static final String TYPE = "es.type";
```

Comment:
If you change this, will it cause compatible issues?

Reply:
type in catalog means that such as es/hive/iceberg, but type in table properties also can be defined. so here i add CATALOG_TYPE = "es.type"